### PR TITLE
docs: add notes about the deployments

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: docs/how-to-enable-new-languages.md [hunk 1]
```diff
@@ -418,4 +418,7 @@ Then commit and push your branch directly to the News repo.
 
 Finally, open a PR for review.
 
-Once both your PRs to the CDN and News repo have been approved, they can be merged. Deployment will be handled subsequently by the staff.
+Once both your PRs to the CDN and News repo have been approved, they can be merged. 
+
+> [!NOTE]
+> Deployment will be handled subsequently by the staff. Here is a sample PR: [freeCodeCamp/news#485](https://github.com/freeCodeCamp/news/pull/485) of how they do it and more details are available in the [staff-wiki](https://staff-wiki.freecodecamp.org/docs/flight-manuals/news-instances#jamstack---news--assets).
```
Comment (ShaunSHamilton) [inline docs/how-to-enable-new-languages.md:424]: I thought staff-wiki was access controlled by `@freecodecamp.org`?

Comment (raisedadead) [reply docs/how-to-enable-new-languages.md:424]: It is; you are welcome to test.

Comment (ShaunSHamilton) [reply docs/how-to-enable-new-languages.md:424]: What I meant is should it be linked to if majority cannot view it?

Comment (raisedadead) [reply docs/how-to-enable-new-languages.md:424]: This is where it is linked to. I am open to ideas though, once you give the current guide a read.

Thanks.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**